pkg/pod/status.go

```diff
@@ -359,6 +334,33 @@ func getWaitingMessage(pod *corev1.Pod) string {
 	return "Pending"
 }
 
+func MarkStatusRunning(trs *v1beta1.TaskRunStatus, reason, message string) {
```

Comment:
as public exported functions, the convention is to include docstrings

Reply:
Thank you for the review. Fixed.